lambda/custom-collection/README.md

```diff
@@ -0,0 +1,86 @@
+# MetaKeep Lambda - Custom ERC721
+
+This repository contains a Custom ERC721 Application that you can use as the starting point
+for using MetaKeep's Lambda Infrastructure.
+
+This project is intended to be used with the
+[MetaKeep Lambda Creation and Invocation Tutorial Series](https://docs.metakeep.xyz/docs/create-your-first-lambda), but you should be
+able to follow it by yourself by reading the README and exploring its
+`contracts` and `scripts` directories.
+
+## Directory Structure
+
+The project is organized as follows:
+
+- [contracts](./contracts): Contains the Smart Contract for Custom ERC721 Tokens.
+- [scripts](./scripts) : Contains the script to deploy custom ERC721 token and whitelist and mint NFT token to the user.
+
+## Quick start
+
+The first thing you need to do is clone this repository:
+
+```sh
+git clone https://github.com/PassbirdCo/MetaKeepCodeSamples.git
+```
+
+Then, on a new terminal, go to the repository's `lambda/custom-collection/smart-contracts` directory and run this to
```

Comment:
rename the folder to custom-erc721

Reply:
done

Reply:
Looks like there are 2 folders custom-nft, custom-collection. There is no custom-erc721 folder

Reply:
Done